Sources/Action.swift

```diff
@@ -10,15 +10,91 @@ import Result
 /// Actions enforce serial execution. Any attempt to execute an action multiple
 /// times concurrently will return an error.
 public final class Action<Input, Output, Error: Swift.Error> {
+	public enum ExecutionStrategy {
+		/// The default execution strategy. Only one worker is allowed to be
+		/// executing at a time. Any further start attempt during the execution
+		/// is rejected with `ActionError.disabled`.
+		case `default`
```

Comment:
I don't think `default` is a great name for this. We should find a name that describes the behavior.